python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst

```diff
@@ -49,15 +46,13 @@ where:
 Let :math:`\cF_i` be the importance factor of the i-th input
 variable, defined by:
 
-  .. math::
-
-     \cF_i = \frac{\left(\frac{\partial h}{\partial x_i} (\muX)\right)^2 \sigma_i^2}{\Var Z}
+.. math::
+    \cF_i = \frac{\left(\frac{\partial h}{\partial x_i} (\muX)\right)^2 \sigma_i^2}{\Var Z}.
```

Comment:
I would rather use $\eta_i$ as in equation 15.11 page 141 in "Sensitivity analysis", Borgonovo, Springer, 2017. This is because caligraphic letters are usually used for a set (e.g. a domain), not for for a scalar.
```suggestion
    \eta_i = \frac{\left(\frac{\partial h}{\partial x_i} (\muX)\right)^2 \sigma_i^2}{\Var Z}.
```